### PR TITLE
Webapp revamp - Landing page

### PR DESCRIPTION
**Pull request by Belco90**
## Changes

- Apply new design to the landing page for mobile and desktop

## Context

Relates to #710

## Tests

I've checked my work by:

- [ ] Adding new tests or adjusting existing tests
- [X] Testing manually

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -30,122 +31,149 @@ import Layout from '~/components/Layout'
...
+    ),
+  },
+]
+
+const FeatureItem = ({ icon, title, children }: FeatureItemProps) => {
+  return (
+    <Box>
+      <Flex
+        boxSize={10}
+        bgColor="blue.200"
```
Comment (HonkingGoose) [inline src/pages/index.tsx:134]:
```suggestion
        bgColor="secondary.200"
```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -30,122 +31,149 @@ import Layout from '~/components/Layout'
...
+        borderRadius="full"
+        alignItems="center"
+        justifyContent="center"
+      >
+        <Icon as={icon} boxSize="18px" color="blue.700" />
```
Comment (HonkingGoose) [inline src/pages/index.tsx:139]:
```suggestion
        <Icon as={icon} boxSize="18px" color="secondary.700" />
```

File: src/pages/index.tsx [hunk 3]
```diff
@@ -30,122 +31,149 @@ import Layout from '~/components/Layout'
...
+      </Flex>
+      <Heading as="h3" color="primaryText" fontSize="2xl" mt={2}>
+        {title}
+      </Heading>
+      <Text
+        fontSize="md"
+        width={{ base: 'full', lg: '70%' }}
+        color="primaryText"
```
Comment (HonkingGoose) [inline src/pages/index.tsx:147]:
```suggestion
        color="secondaryText"
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Changes so our code matches the Figma design:

- Use `secondary.*` as color token for the icon and background
- Use `secondaryText` for the supporting text for each feature

File: src/pages/index.tsx (hunk 1 above)
Comment (Belco90) [reply src/pages/index.tsx:134]: Damn! Good catch, I'll address these changes later. Thanks!

File: src/pages/index.tsx [hunk 4]
```diff
@@ -30,122 +31,149 @@ import Layout from '~/components/Layout'
...
+    ),
+  },
+]
+
+const FeatureItem = ({ icon, title, children }: FeatureItemProps) => {
+  return (
+    <Box>
+      <Flex
+        boxSize={10}
+        bgColor="secondary.200"
```
Comment (HonkingGoose) [inline src/pages/index.tsx:134]: On Figma, the dark mode:

- uses `secondary.800` for the round background fill
- uses `secondary.200` for the icon itself

Right now the dark mode has the same icons as the light mode, which makes them "vibrate" when I look at them. 😄

Can you try to fix this?

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
This is getting really similar to what I designed, but there's still a big difference with the icons in the dark mode. 😉

File: src/pages/index.tsx (hunk 4 above)
Comment (Belco90) [reply src/pages/index.tsx:134]: Sure! I completely forgot about the dark mode, I'll fix it later.

Comment (HonkingGoose) [reply src/pages/index.tsx:134]: Cool!

Do you want to merge what we have already? You can then work on the dark mode changes in a separate PR?

Or do you want to do all the frontpage work at once in this PR?

**Comment by Belco90** [comment]
@HonkingGoose feel free to approve the snapshots in Happo if you are happy with them!

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**